__tests__/app.test.js

```diff
@@ -246,6 +246,84 @@ describe("my express project", () => {
           expect(articles).toBeSortedBy("created_at", { descending: true });
         });
     });
+
+    it("200 sort_by, which sorts the articles by any valid column (defaults to date)", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("votes", { descending: true });
+        });
+    });
+
+    it("200 order, which sorts the articles by any valid column (defaults to date)order(defaults to descending)", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title&order=asc")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("title");
+        });
+    });
+
```

Comment:
these changes are in the wrong pull request

please keep code on the correct branch!